llama-index-integrations/vector_stores/llama-index-vector-stores-kdbai/llama_index/vector_stores/kdbai/base.py

```diff
@@ -0,0 +1,214 @@
+"""KDB.AI vector store index.
+
+An index that is built within KDB.AI.
+
+"""
+
+import logging
+from typing import Any, List, Callable, Optional
+
+import pandas as pd
+
+from llama_index.core.bridge.pydantic import PrivateAttr
+from llama_index.core.schema import BaseNode, TextNode
+from llama_index.core.vector_stores.types import (
+    BasePydanticVectorStore,
+    VectorStoreQuery,
+    VectorStoreQueryResult,
+)
+from .utils import default_sparse_encoder
```

Comment:
lets use the full import here `llama_index.vector_stores.kbdai.utils`

Reply:
Updated